lib/src/CoE/OD.cc

```diff
@@ -306,4 +313,13 @@ namespace kickcat::CoE
         static Dictionary dict;
         return dict;
     }
+ 
+ 	void addEntryString(Object &object, uint8_t subindex, uint16_t bitlen, uint16_t access,
+              DataType type, std::string const& description, std::string data)
+    {
+        object.entries.emplace_back(subindex,bitlen,access,type,description);
+        std::string* dataAlloc = new std::string(data);
```

Comment:
Copy the content instead of the object to avoid leak if the entry is deleted (through free, not delete)